lib/shoryuken/cli.rb

```diff
@@ -41,7 +41,7 @@ def run(args)
       @launcher = Shoryuken::Launcher.new
 
       if callback = Shoryuken.start_callback
-        logger.info "Calling Shoryuken.on_start block"
+        logger.info { "Calling Shoryuken.on_start block" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.